### DISCUSSION TITLE
Will vidstack have support for keyboard gestures?

### DISCUSSION THREAD

**Discussion by rosnerdev** [vidstack/player#671] [Q&A] [answered]
For example on youtube when you click `k` it stops the video or `m` for muting.

**Comment by mihar-22** [comment, marked as answer]
Yep! We'll need to design a keyboard interface so you can assign keys to any control. The default player skin will come with preconfigured keys similar to popular players like YouTube.

**Comment by mihar-22** [comment]
Update here that keyboard gestures are now supported: https://www.vidstack.io/docs/player/core-concepts/keyboard 🥳
